Docs/source/install/hpc/narval.rst

```diff
@@ -0,0 +1,153 @@
+.. _building-narval:
+
+Narval (Digital Research Alliance of Canada)
+========================================================
+
+The `Narval cluster <https://www.calculquebec.ca/en/nouvelle/narval-a-new-supercomputer-dedicated-to-scientific-research/>`__ is located at McGill University.
+
+Introduction
+------------
+
+If you are new to this system, **please see the following resources**:
+
+* `Narval Wiki <https://docs.alliancecan.ca/wiki/Narval/en>`__
+* `Infrastructures and Services <https://www.calculquebec.ca/en/academic-research-services/infrastructures-services/#ouvert>`__
+
+Through Narval we have access to CPU and GPU nodes.
+
+Installation
+------------
+
+Narval has no WarpX preinstalled, we need to compile it from scratch. Since ``$HOME`` directory is designed for compiling codes, the following operations will be executed in ``$HOME`` directory.
+
+Download WarpX
+^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    cd $HOME
+    git clone https://github.com/ECP-WarpX/WarpX.git
+
+Building WarpX
+^^^^^^^^^^^^^^
+
+* To build it with CPU backend, load the following module
+
+.. code-block:: bash
+
+    module load StdEnv/2023
+
+This environment contains ``gcc/12.3, openmpi/4.1.5`` and other usefule modules. Now build WarpX as described in `documentation <https://warpx.readthedocs.io/en/latest/install/cmake.html>`__
```

Comment:
Thank you! In a follow-up, can you add this in our usual `warpx.profile` style scripts as in other HPC systems (e.g., Perlmutter)? That makes it easier to add more options (such as project ids, aliases to get interactive nodes) and keeps it consistent for users switching systems.